"JWT token" is redundant. Removed instances of 'token' after "JWT" (#1328)
OWASP/CheatSheetSeries: cheatsheets/HTML5_Security_Cheat_Sheet.md

When using websocket as communication channel, it's important to use an authentication method allowing the user to receive an access *Token* that is not automatically sent by the browser and then must be explicitly sent by the client code during each exchange.

> HMAC digests are the simplest method, and [JSON Web Token](https://jwt.io/introduction/) is a good feature rich alternative, because it allows the transport of access ticket information in a stateless and not alterable way. Moreover, it defines a validity timeframe. You can find additional information about JWT hardening on this [cheat sheet](JSON_Web_Token_for_Java_Cheat_Sheet.md).

Reviewer: ```suggestion
HMAC digests are the simplest method and [JSON Web Tokens](https://jwt.io/introduction/) (JWT) are a good feature rich alternative, because they allow the transport of access ticket information in a stateless and unalterable way. Moreover, they define a validity timeframe. You can find additional information about JWT hardening on this [cheat sheet](JSON_Web_Token_for_Java_Cheat_Sheet.md).
```
